### ISSUE TITLE
Support gitmoji

### ISSUE THREAD

**Issue by MarcusOtter** [Nutlope/aicommits#20]
It would be cool if it could optionally automatically pick a nice emoji, for example following https://gitmoji.dev

**Comment by AntoineKM** [comment]
ahah I was going to make the same suggestion

**Comment by Nutlope** [comment]
Agreed! Added this feature suggestion to the README and will plan to implement it under a feature flag so only those who want it can use it

**Comment by privatenumber** [comment]
Would be helpful if someone can engineer a prompt for this and suggest it here. You can experiment for free on https://chat.openai.com.

**Comment by kcoderhtml** [comment]
A variation of this might work.
```
I will give you a commit message for git, and you will classify the commit category and pick a suitable gitmoji for the category. Respond with only the emoji and then the category you classified the commit as, don't give any commentary. Here is your first commit: {{commitMessage}}
```

**Comment by kcoderhtml** [comment]
Example output:
feat: show multiple options (#64) > :sparkles: feat (feature)
chore: add commit hooks (#56) > :wrench: chore (maintenance)
Merge pull request #63 from Nutlope/bundle-deps > :arrows_counterclockwise: merge (merge)

**Comment by kcoderhtml** [comment]
v2 that only gives the text code:
```
I will give you a commit message for git, and you will classify the commit category and pick a suitable gitmoji for the category.
Respond with only the gitmoji text code and not an emoji or commit catagory, don't give any commentary.
Response should be formated like this without the quotes: ':sparkles:'
Here is your commit:  {{commitMessage}}
```

**Comment by Th3Kr4t3** [comment]
I sent a PR on how to implement this, I added a prompt to the end like this:
`preface with the suitable Gitmoji markup between colons:` 
to use it:
> Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff preface with the suitable Gitmoji markup between colons:\n${diff}

**Comment by kud** [comment]
No update about it? Can't wait to get it.
